librz/core/meson.build

```diff
@@ -60,6 +60,7 @@ rz_core_sources = [
   #'rtr_shell.c',
   'seek.c',
   'serialize_core.c',
+  'serialize_core_seek.c',
```

Comment:
The seek serialization can be put into `serialize_core.c` too so it is all together.